vault/tests/docker/metrics_policy.hcl

```diff
@@ -1,19 +1,3 @@
 path "sys/metrics*" {
   capabilities = ["read", "list"]
 }
-
-path "sys/audit" {
-  capabilities = ["read", "sudo"]
-}
-
-path "sys/license" {
-  capabilities = ["read", "list"]
-}
-
-path "sys/mounts" {
-  capabilities = ["read", "list"]
-}
-
-path "sys/host-info" {
-  capabilities = ["read", "list"]
-}
```

Comment:
note: those seems not necessary.

Reply:
I'd keep license for when we start monitoring that

Reply:
Sure, I can add the `sys/licence` back.

Caveat, `unauthenticated_metrics_access` **won't** work for data not part of `sys/metrics`.